### DISCUSSION TITLE
When opening a comic is it possible to have archive overlay over the homescreen/searched results instead

### DISCUSSION THREAD

**Discussion by HiddenMelonCat** [Difegue/LANraragi#783] [Q&A] [answered]
To give a more in-depth explanation. Let's say I click on a comic. It will open that comic then it will have the archive overlay on top of it. What I want instead is that when I click on a comic it has the homepage and or searched results in the background and the overlay on top of that. That way if I change my mind I can just click the background and pick a different comic or go to search for something new, or if I want start reading I would then just click on the first image. 

How it is setup now it comes off as redundant to me. Which is opening the gallery to read then opening the archive overlay on top of it. 
If there is a way to change it to my preferred viewing way then that would be appreciated greatly. If not then I guess you can view this as a feature suggestion.  Thank you! Before writing this I have searched with the keyword "overlay" and got no sufficient results. I am new to github though, so excuse any errors in my ways.

**Comment by Difegue** [comment, marked as answer]
👋 Currently there's no way to show an archive overlay in the index, since generating all the page thumbnails the first time a comic is opened is a fairly expensive operation.  

I can recommend opening comics in new tabs instead if you want a similar-ish behavior for now, although I understand it can be annoying to do on mobile devices.. I'll keep this ask in mind though, thanks for the feedback!
